File: java/jse-workspace/fundamentos/src/com/pablomonteserin/clasesabstractasinterfaces/claseabstracta/Vaca.java
package com.pablomonteserin.clasesabstractasinterfaces.claseabstracta;

public class Vaca extends Animal {

	Vaca(String sonido) {
		super(sonido);
	}

	@Override
	public void comunicarse() {
		System.out.println("La vaca cuando está contenta hace " + sonido);
	}
	
	

}

File: c-sharp/ex/constructors/heritage/exercises/animal-inheritance/Vaca.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AnimalInheritance
{
  class Vaca : Animal
  {
    int litrosLeche;

    public Vaca(string sonido) : base(sonido) {}

    public void Comunicarse()
    {
      Console.WriteLine("Cuando la vaca tiene leche hace " + sonido + "- " + litrosLeche);
    }

  }
}


